README.md

```diff
@@ -1026,6 +1027,43 @@ datetime                   inExchanges  inTopHoldersTotal  outsideExchanges
+
+```
+[{'datetime': '2019-04-21T13:51:59.000000Z', 'fromAddress': {'address': '0x1f3df0b8390bb8e9e322972c5e75583e87608ec2', 'isExchange': False}, 'toAddress': {'address': '0x5eaae5e94995202c2183956369b1700450b2c89e', 'isExchange': False}, 'trxHash': '0xdbced935b09dd0b9747d004f6242eb1e31b4770ae994d15ece8ebb27f3b81524', 'trxValue': 166673.99999999997}, {'datetime': '2019-04-28T07:43:38.000000Z', 'fromAddress': {'address': '0x0a920bfdf7f9777a32699aeb7a44d857c1ad7d1e', 'isExchange': False}, 'toAddress': {'address': '0x868074aab18ea32e0103e9b3c10cc0a958eeb324', 'isExchange': False}, 'trxHash': '0x5f2214d34bcdc3298799c1a58ee73e4d95ee94310e7873760b1b64ee114e0f90', 'trxValue': 33181.82278963}, {'datetime': '2019-04-28T07:53:32.000000Z', 'fromAddress': {'address': '0x868074aab18ea32e0103e9b3c10cc0a958eeb324', 'isExchange': False}, 'toAddress': {'address': '0x876eabf441b2ee5b5b0554fd502a8e0600950cfa', 'isExchange': True}, 'trxHash': '0x90bd286da38a2bf6de0aa2415e74bfd1400982168e9d7339021828eccccd90f8', 'trxValue': 33181.82278963}, {'datetime': '2019-04-26T14:38:45.000000Z', 'fromAddress': {'address': '0x876eabf441b2ee5b5b0554fd502a8e0600950cfa', 'isExchange': True}, 'toAddress': {'address': '0x76af586d041d6988cdba95347e2f857872524fea', 'isExchange': False}, 'trxHash': '0xe45b86f415e9300afb1caa2f2abcb5e965f22acc617a0658264608cdc8529b50', 'trxValue': 28999.640230349996}, {'datetime': '2019-04-30T15:17:28.000000Z', 'fromAddress': {'address': '0x876eabf441b2ee5b5b0554fd502a8e0600950cfa', 'isExchange': True}, 'toAddress': {'address': '0x1f4a90043cf2de10df66b6ba46c824ea7e27b34e', 'isExchange': False}, 'trxHash': '0xc85892b9ef8c647957583cb9fb1f44a7dd89ee91410d83b266eec34c589fd875', 'trxValue': 20544.42975}]
+```
+
+
+```python
+san.get(
+    "token_top_transactions/santiment",
+    from_date="2019-04-18",
+    to_date="2019-04-30",
+    limit=5,
+)
+```
+
+Example result:
+
+```
+datetime                                                  fromAddress  fromAddressInExchange  ...                                            trxHash      trxValue                                              ...                                                                 
+2019-04-21 13:51:59+00:00  0x1f3df0b8390bb8e9e322972c5e75583e87608ec2                  False  ...  0xdbced935b09dd0b9747d004f6242eb1e31b4770ae994...  166674.00000
+2019-04-28 07:43:38+00:00  0x0a920bfdf7f9777a32699aeb7a44d857c1ad7d1e                  False  ...  0x5f2214d34bcdc3298799c1a58ee73e4d95ee94310e78...   33181.82279
+2019-04-28 07:53:32+00:00  0x868074aab18ea32e0103e9b3c10cc0a958eeb324                  False  ...  0x90bd286da38a2bf6de0aa2415e74bfd1400982168e9d...   33181.82279
+2019-04-26 14:38:45+00:00  0x876eabf441b2ee5b5b0554fd502a8e0600950cfa                   True  ...  0xe45b86f415e9300afb1caa2f2abcb5e965f22acc617a...   28999.64023
+2019-04-30 15:17:28+00:00  0x876eabf441b2ee5b5b0554fd502a8e0600950cfa                   True  ...  0xc85892b9ef8c647957583cb9fb1f44a7dd89ee91410d...   20544.42975
+```
```

Comment:
the `toAddress` data is missing. Also shorten the addresses and the hash like this:
```
datetime                          fromAddress    fromAddressIsExchange            trxHash      trxValue
2019-04-21 13:51:59+00:00  0x1f3df0b8390bb8...                   False   0xdbced935b09dd0  166674.00000
2019-04-28 07:43:38+00:00  0x0a920bfdf7f977...                   False   0x5f2214d34bcdc3   33181.82279
2019-04-28 07:53:32+00:00  0x868074aab18ea3...                   False   0x90bd286da38a2b   33181.82279
2019-04-26 14:38:45+00:00  0x876eabf441b2ee...                    True   0xe45b86f415e930   28999.64023
2019-04-30 15:17:28+00:00  0x876eabf441b2ee...                    True   0xc85892b9ef8c64   20544.42975
```

Reply:
You're right, I've fixed the README. It should look normal now.